tox.ini

```diff
@@ -28,7 +28,7 @@ commands =
 [testenv:flake8]
 skip_install = true
 deps =
-    flake8==3.4.1
+    flake8==3.7.7
```

Comment:
`flake8` is now at version `3.7.9`. Should we leave it at 3.7.7?

Reply:
No, my bad. Updating.